client/src/Components/CreateTask.js

```diff
@@ -83,6 +82,13 @@ export default function CreateTask() {
                 id="description"
                 onChange={(e) => setDescription(e.target.value)}
               />
+              <div>
+                {errors.map((error, index) => (
+                  <div key={index} className="text-danger">
+                    {error.path === "description" && error.msg}
```

Comment:
When "error.path" not equal to "description" then only empty div will show, bad practice